pandas/io/excel/_base.py

```diff
@@ -103,7 +103,12 @@
     of dtype conversion.
 engine : str, default None
     If io is not a buffer or path, this must be set to identify io.
-    Acceptable values are None, "xlrd", "openpyxl" or "odf".
+    Acceptable values are None, "xlrd", "openpyxl", "odf", or "pyxlsb".
+    Engine compatibility:
+    "xlrd" supports most old/new Excel file formats.
```

Comment:
I think you can use bullets here (e.g. `-`) to more doc-string clarify

Reply:
will do

Reply:
I think it's clearer now. Thank you for the suggestion. Same for the changes proposed below.